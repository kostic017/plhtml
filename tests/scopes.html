<!doctype html>
<html lang="en">
    <head>
        <title>`Scopes Test`</title>
    </head>
    <body>

        <main>

            <var class="integer">a</var>
            <data value="1">a</data>
            <output>a + `\n`</output>

            <div data-if="true">

                <div data-if="true">

                    <var class="integer">a</var>
                    <data value="3">a</data>
                    <output>a + `\n`</output>

                </div>

                <output>a + `\n`</output>

                <var class="integer">a</var>
                <data value="2">a</data>
                <output>a + `\n`</output>

            </div>

        </main>

    </body>
</html>
