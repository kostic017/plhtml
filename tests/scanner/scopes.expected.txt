<
!
Doctype
Html
>
<
Html
Lang
=
"
Identifier|en|
"
>
<
Head
>
<
Title
>
StringConst|Scopes Test|
<
/
Title
>
<
/
Head
>
<
Body
>
<
Main
>
<
Var
Class
=
"
Identifier|integer|
"
>
Identifier|a|
<
/
Var
>
<
Data
Value
=
"
IntConst|1|
"
>
Identifier|a|
<
/
Data
>
<
Output
>
Identifier|a|
+
StringConst|\n|
<
/
Output
>
<
Div
Data
-
If
=
"
BoolConst|true|
"
>
<
Div
Data
-
If
=
"
BoolConst|true|
"
>
<
Var
Class
=
"
Identifier|integer|
"
>
Identifier|a|
<
/
Var
>
<
Data
Value
=
"
IntConst|3|
"
>
Identifier|a|
<
/
Data
>
<
Output
>
Identifier|a|
+
StringConst|\n|
<
/
Output
>
<
/
Div
>
<
Output
>
Identifier|a|
+
StringConst|\n|
<
/
Output
>
<
Var
Class
=
"
Identifier|integer|
"
>
Identifier|a|
<
/
Var
>
<
Data
Value
=
"
IntConst|2|
"
>
Identifier|a|
<
/
Data
>
<
Output
>
Identifier|a|
+
StringConst|\n|
<
/
Output
>
<
/
Div
>
<
/
Main
>
<
/
Body
>
<
/
Html
>
